conda-requirements.txt

```diff
@@ -1,10 +1,10 @@
 nomkl==1.0
 bokeh==0.12.7
 dill==0.2.6
-flask==0.12.2
+flask==0.12.3
```

Comment:
conda does not have this in their channels, we can either use pip or move to 1.0.2

Reply:
I went with pip.